python/tidy/servo_tidy/tidy.py

```diff
@@ -694,6 +698,46 @@ def check_spec(file_name, lines):
                 brace_count -= 1
 
 
+def check_config_file(config_file, print_text=True):
+    if conf_file and config_file == CONFIG_FILE_PATH:
+        lines = conf_file.splitlines(True)
+    else:
+        # Needed for running tests
+        with open(config_file, "r") as content:
+            lines = content.read().splitlines(True)
+
+    if print_text:
+        print '\rChecking config file for tidiness...'
```

Comment:
We're not tidying the config file, are we? So, this should just be `"Checking the config file..."`